pom.xml

```diff
@@ -61,7 +61,7 @@
         <dependency>
             <groupId>org.apache.guacamole</groupId>
             <artifactId>guacamole-ext</artifactId>
-            <version>1.0.0</version>
+            <version>1.2.0</version>
```

Comment:
If using a different version of `guacamole-ext`, the API version noted within `guac-manifest.json` should also be updated:

https://github.com/glyptodon/guacamole-auth-json/blob/f7b2eaf6a65b7cd25fd73437360e36fe46e0bcb9/src/main/resources/guac-manifest.json#L3

It will otherwise be possible to load this extension in a version of Guacamole that does not use the expected API.